step.go

```diff
@@ -24,15 +24,25 @@ func (s *stepObject) AddStep(step stepObject) {
 	s.ChildrenSteps = append(s.ChildrenSteps, step)
 }
 
+// allure.Step is meant to be wrapped around actions
```

Comment:
The comment must always start by the name of the method. In this case:
// Step is meant .....